game.cpp

```diff
@@ -75,10 +75,6 @@ double calc_mean(const std::vector<double>& v)
   ) / v.size();
 }
 
-double get_nth_player_diameter(const game &in_game, const int &id)
-{
-    return in_game.get_player(id).get_diameter();
-}
 double get_nth_player_size(const game& g, const int i)
```

Comment:
A bit of an unexpected choice: reserve `get_nth_player_size` over `get_nth_player_diameter` when the player has a member function called 'get_diameter'.

Still, I am happy to have gotten rid of the duplicate :-)